problem.py

```diff
@@ -12,12 +12,12 @@
 # An object implementing the workflow
 workflow = rw.workflows.ObjectDetector()
 
-minipatch = [56, 168, 56, 168]
+minipatch = [28, 196, 28, 196]
```

Comment:
Maybe write a comment to explain what these values correspond to. 
Something like 
```
# The overlap between adjacent patches is 56 pixels
# The scoring region is chosen so that despite the overlap, 
# no crater is scored twice, hence the boundaries of 
# 28 = 56 / 2 and 196 = 224 - 56 / 2
```